src/models/cart-model.js

```diff
@@ -96,6 +96,16 @@ const CartModel = BaseModel.extend({
   },
 
   /**
+    * Returns URI to an image that can be used as default image when a variant does
+    * not have an image. i.e when `variant.image` yields `undefined`
+    * @property noImageURI
+    * @type {String}
+  */
+  get noImageURI() {
+    return 'https://widgets.shopifyapps.com/assets/no-image.svg';
```

Comment:
So, this image is for products. It should be exported by the product module, not a getter on the cart. In globals mode, it should be an attribute on the global (`shopify.js`). It's also a constant, so it should be all caps. Not a getter.